ppr-ui/src/components/common/DocumentId.vue

```diff
@@ -83,35 +80,50 @@ export default defineComponent({
       loadingDocId: false,
       isUniqueDocId: false,
       displayDocIdError: false,
-      validateDocId: computed(() => props.validate && !localState.isVerifiedDocId),
+      showBorderError: computed(() => props.validate && !localState.isVerifiedDocId),
       isVerifiedDocId: computed(() => {
         return localState.isDocumentIdFormValid && localState.isUniqueDocId
       }),
       uniqueDocIdError: computed(() => {
         // Manual error handling for Unique DocId Lookup
-        return localState.displayDocIdError ? ['Must be unique number'] : []
+        return localState.displayDocIdError ? ['Must be unique number'] : ''
+      }),
+      documentIdRules: computed(() => {
+        return props.validate
+          ? customRules(
+            required('Enter a Document ID'),
+            maxLength(8, true),
+            minLength(8, true),
+            isNumber()
+          )
+          : customRules(
+            maxLength(8, true),
+            isNumber()
+          )
       })
     })
 
-    const documentIdRules = customRules(
-      required('Enter a Document ID'),
-      maxLength(8, true),
-      isNumber()
-    )
-
-    watch(() => props.validate, async () => {
+    const validateForm = () => {
       documentIdForm.value?.validate()
+    }
```

Comment:
Is this function needed considering its one line?

Reply:
Would probably second that, unless the intention is to expose the `validateForm` function for parent use.

Reply:
Cleaned up.